### DISCUSSION TITLE
稳定使用了8天，再次发生未知错误了

### DISCUSSION THREAD

**Discussion by kevzcr** [shuax/MouseInc.Settings#379] [General]
8月29号之前出过一次这样的问题，反馈后解决了，用到今天，再次出现发生未知错误，我把OCR的3个接口来回切换过了数次，没一个能用，想了想，好像也没动什么设置，没见有人反馈，可能是自己的问题，挺好的一个功能，让我玩坏了。关键是还不知道乍解决。

**Comment by war525** [comment]
不是哦，是真不能用哦，目前最新版的不能用，本地有一个老版本可用

**Reply by kevzcr** [reply]
你好，是哪个版本，有链接吗？

**Reply by mMrWang** [reply]
OCR接口咋换兄弟

**Reply by war525** [reply]
2.12.1

**Reply by kevzcr** [reply]
> OCR接口咋换兄弟

![image](https://user-images.githubusercontent.com/88956040/188847535-5508f294-52a1-4119-bed1-4f782de268f5.png)

**Reply by kevzcr** [reply]
> 不是哦，是真不能用哦，目前最新版的不能用，本地有一个老版本可用

下载了你说的这个版本，打不开软件的设置界面。

**Comment by 6Tom** [comment]
有道和飞浆的接口没有问题。。。你们是不是乱用翻墙软件了，试下重置代理

以下“重置代理.bat”
```
cd /d "%~dp0"
set "reg_=HKCU\SOFTWARE\Microsoft\Windows\CurrentVersion\Internet Settings\Connections"
set "value_=4600000002000000090000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
reg export "%reg_%" "Connections_bak.reg" /y
reg add "%reg_%" /v "DefaultConnectionSettings" /t REG_BINARY /d "%value_%" /f
reg add "%reg_%" /v "SavedLegacySettings" /t REG_BINARY /d "%value_%" /f
```

**Reply by kevzcr** [reply]
感谢`6Tom`，照着做了，`自用`不行，`有道`不行，但`飞桨`可以用了。就知道问题出在自己身上，只是不会解决，还好有大佬相助，谢谢拉。
也期望有人能告诉我，其他2种方式还有救吗？
